Dockerfile

```diff
@@ -0,0 +1,16 @@
+#Pull slim Python image
+FROM python:3.10-slim-buster
+
+# Set the working directory
+WORKDIR /eetc_kraken
+
+# Copy requirements.txt and install dependencies
+COPY requirements.txt requirements.txt
+RUN pip3 install -r requirements.txt
+
+# Copy all required files
+COPY . .
```

Comment:
```suggestion
Copy all required files
COPY . .

# Install dependencies
RUN pip3 install -r requirements.txt
```

Reply:
https://stackoverflow.com/questions/72113469/why-are-python-project-files-copied-after-installing-requirements-in-dockerfile
https://docs.docker.com/language/python/build-images/